python/python/bystro/search/index/listener.py

```diff
@@ -14,33 +14,12 @@
 from bystro.beanstalkd.worker import ProgressPublisher, QueueConf, listen
 from bystro.search.utils.annotation import get_config_file_path
 from bystro.search.utils.messages import IndexJobCompleteMessage, IndexJobData, IndexJobResults
-from bystro.utils.config import _get_bystro_project_root
 
 from msgspec import json
 
 TUBE = "index"
 
-_GO_HANDLER_BINARY_PATH: str | None = None
```

Comment:
Relying on a binary path to go/cmd/opensearch was incompatible with installing the tool into the user's path. This bug bit me when deploying to IBDGC.